### DISCUSSION TITLE
Support m x n seed table

### DISCUSSION THREAD

**Discussion by forhappy** [mdx-editor/editor#156] [Ideas]
Can we pass m x n parameters to the seedTable() in table plugin so that user can insert tables that contain a few predefined dimensions?

**Comment by petyosi** [comment]
Extending the initial table creation (as in google docs, for example), is part of my plans, but I don't have any concrete timeline for it. Happy to accept a contribution.

**Reply by forhappy** [reply]
Create a PR to support insert m x n table: https://github.com/mdx-editor/editor/pull/160
